target/linux/ramips/dts/mt7620a_asus_rt-ac51u.dts

```diff
@@ -114,6 +103,16 @@
 	status = "okay";
 };
 
+&gpio0 {
+	status = "okay";
+	enable-leds {
+		gpio-hog;
+		line-name = "enable-leds";
+		output-low;
```

Comment:
Have you tested this?

Reply:
yes, is currently working on my desk. 😉

Reply:
Okay, because I would have expected output-high. But since it's working, I must have done a 0/1 wrong somewhere.

Reply:
I am still learning a lot, in fact I had this working by trial and error, and probably I still don't get the difference between
`gpios = <10 GPIO_ACTIVE_HIGH>;`
and
`output-low;`
Reading [here](https://www.kernel.org/doc/Documentation/devicetree/bindings/gpio/gpio.txt)
seems that `<10 GPIO_ACTIVE_HIGH>` gives the instruction to the pin 10 to be "ON" if the cpu gives "HIGH" voltage (3v or 5v i think). So in this case when I set `output-low`, am I forcing it "OFF". Is this correct?

Reply:
@DavideFioravanti

For `GPIO_ACTIVE_HIGH` is a consumer flag, which specifies if the consumer is active when the pin is high (1). `GPIO_ACTIVE_LOW` is the other way around (consumer active when the pin is low).

However, as you are using it in the context of a GPIO hog, the GPIo flag has no influence on the state of the Hog. The state of the pin is exclusively set using the `output-low` or `output-high` property. It has to be defined, as the `#gpio-cells` property in the [GPIO controller node](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7620a.dtsi#L116) is set to 2. Without the consumer flag, the device-tree won't compile.

Reply:
Understood. Thank you very much for the explanation 😃